contrib/devtools/optimize-pngs.py

```diff
@@ -37,7 +37,7 @@ def content_hash(filename):
     for file in os.listdir(absFolder):
         extension = os.path.splitext(file)[1]
         if extension.lower() == '.png':
-            print("optimizing "+file+"..."),
+            print("optimizing "+file+"...")
```

Comment:
Trailing comma in python2 outputs space instead of newline. I think equivalent in python3 would be 

```python
print("optimizing {}...".format(file), end=' ')
```

Reply:
thanks. Fixed

Reply:
Was this fixed?